modules/openapi-generator/src/main/resources/JavaSpring/apiController.mustache

```diff
@@ -109,7 +111,10 @@ public class {{classname}}Controller implements {{classname}} {
     {{#isDeprecated}}
     @Deprecated
     {{/isDeprecated}}
-    public {{#responseWrapper}}{{.}}<{{/responseWrapper}}ResponseEntity<{{>returnTypes}}>{{#responseWrapper}}>{{/responseWrapper}} {{operationId}}(
+    {{^useResponseEntity}}
+    @ResponseStatus({{#springHttpStatus}}{{{code}}}{{/springHttpStatus}})
```

Comment:
not required here since it's present in interface

Reply:
bellow there is one more place where `ResponseEntity` is used (Can't put a comment at that line)

Reply:
Oh, you're right.